src/components/Card/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
```

Comment:
The Card component is very clever and useful. This component will allow us to quickly add content to subsequent pages in a uniform manner.